src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Providers/Providers/AzureFilesPsBackupProvider.cs

```diff
@@ -428,12 +431,135 @@ public List<RecoveryPointBase> ListRecoveryPoints()
 
         public ProtectionPolicyResource CreatePolicy()
         {
-            throw new NotImplementedException();
+            string vaultName = (string)ProviderData[VaultParams.VaultName];
```

Comment:
Can we merge the CreatePolicy and MergePolicy methods? This can be achieved by introducing a CreateOrUpdatePolicy helper method which both the CreatePolicy and ModifyPolicy can call separately.